compiler/luci-interpreter/src/kernels/Softmax.test.cpp

```diff
@@ -26,17 +26,35 @@ namespace
 
 using namespace testing;
 
-TEST(SoftmaxTest, Float)
+template <typename T>
+void Check(std::initializer_list<int32_t> input_shape, std::initializer_list<int32_t> output_shape,
+           std::initializer_list<float> input_data, std::initializer_list<float> output_data)
 {
-  Shape input_shape{2, 1, 2, 3};
-  std::vector<float> input_data{
-      5,  -9, 8,  //
-      -7, 2,  -4, //
-      1,  -2, 9,  //
-      3,  -6, -1, //
-  };
-  Tensor input_tensor = makeInputTensor<DataType::FLOAT32>(input_shape, input_data);
-  Tensor output_tensor = makeOutputTensor(DataType::FLOAT32);
+  float kQuantizedTolerance = getTolerance(std::min<float>(std::min<float>(output_data), 0.f),
+                                           std::max<float>(std::max<float>(output_data), 0.f), 255);
+  std::pair<float, int32_t> input_quant_param =
+      quantizationParams<T>(std::min<float>(std::min<float>(input_data), 0.f),
+                            std::max<float>(std::max<float>(input_data), 0.f));
+  std::pair<float, int32_t> output_quant_param =
+      quantizationParams<T>(std::min<float>(std::min<float>(output_data), 0.f),
+                            std::max<float>(std::max<float>(output_data), 0.f));
+  Tensor input_tensor{getElementType<T>(),
+                      input_shape,
+                      {{input_quant_param.first}, {input_quant_param.second}},
+                      ""};
+  if (std::is_floating_point<T>::value)
+  {
+    input_tensor.writeData(input_data.begin(), input_data.size() * sizeof(T));
```

Comment:
Please use `makeInputTensor` (see #4198 for reference) <3

Reply:
as same as above.